modules/_labs/select/react/lib/Select.tsx

```diff
@@ -0,0 +1,681 @@
+import * as React from 'react';
+import {styled, Themeable} from '@workday/canvas-kit-labs-react-core';
+import {GrowthBehavior, ErrorType, errorRing} from '@workday/canvas-kit-react-common';
+import {CSSObject} from '@emotion/core';
+import {
+  colors,
+  borderRadius,
+  inputColors,
+  spacingNumbers,
+  type,
+  spacing,
+} from '@workday/canvas-kit-react-core';
+import {caretDownSmallIcon} from '@workday/canvas-system-icons-web';
+import {SystemIcon} from '@workday/canvas-kit-react-icon';
+import {SelectMenu, menuFadeDuration} from './SelectMenu';
+import {SelectOptionProps} from './SelectOption';
+import uuid from 'uuid/v4';
+
+export interface SelectProps
+  extends Themeable,
+    GrowthBehavior,
+    Pick<React.InputHTMLAttributes<HTMLInputElement>, 'onChange'>,
+    Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'onChange'> {
+  /**
+   * The SelectOption children of the Select.
+   */
+  children?: React.ReactElement<SelectOptionProps>[];
+  /**
+   * The type of error associated with the Select (if applicable).
+   */
+  error?: ErrorType;
+  /**
+   * The value of the Select.
+   */
+  value?: string;
+}
+
+export interface SelectState {
+  focusedOptionIndex: number;
+  isMenuHidden: boolean;
+  isMenuHiding: boolean;
+  label: string;
+  selectedOptionIndex: number;
+}
+
+const focusButtonCSS = (): CSSObject => ({
```

Comment:
Nit: This doesn't need to be a function, it can just be a regular object
```suggestion
const focusButtonCSS = {
```

Reply:
Fixed